Symfony/CS/Console/Output/ProcessOutput.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Console\Output;
+
+use Symfony\Component\Console\Output\StreamOutput;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+use Symfony\CS\FixerFileProcessedEvent;
+
+/**
+ * Output writer to show the process of a FixCommand.
+ *
+ * @internal
+ */
+class ProcessOutput
+{
+    /**
+     * File statuses map.
+     *
+     * @var array
+     */
+    private static $eventStatusMap = array(
+        FixerFileProcessedEvent::STATUS_UNKNOWN => array('symbol' => '?', 'format' => '%s', 'description' => 'unknown'),
+        FixerFileProcessedEvent::STATUS_INVALID => array('symbol' => 'I', 'format' => '<bg=red>%s</bg=red>', 'description' => 'invalid file syntax, file ignored'),
+        FixerFileProcessedEvent::STATUS_SKIPPED => array('symbol' => 'S', 'format' => '<fg=cyan>%s</fg=cyan>', 'description' => 'Skipped'),
+        FixerFileProcessedEvent::STATUS_NO_CHANGES => array('symbol' => '.', 'format' => '%s', 'description' => 'no changes'),
+        FixerFileProcessedEvent::STATUS_FIXED => array('symbol' => 'F', 'format' => '<fg=green>%s</fg=green>', 'description' => 'fixed'),
+        FixerFileProcessedEvent::STATUS_EXCEPTION => array('symbol' => 'E', 'format' => '<bg=red>%s</bg=red>', 'description' => 'error'),
+        FixerFileProcessedEvent::STATUS_LINT => array('symbol' => 'E', 'format' => '<bg=red>%s</bg=red>', 'description' => 'error'),
+    );
+
+    /**
+     * Event dispatcher instance.
+     *
+     * @var EventDispatcher
+     */
+    private $eventDispatcher;
+
+    /**
+     * Stream output instance.
+     *
+     * @var StreamOutput
+     */
+    private $output;
+
+    /**
+     * File pointer of type stream.
+     *
+     * @var resource
+     */
+    private $stdErr;
+
+    public function __construct(EventDispatcher $dispatcher)
+    {
+        $this->stdErr = fopen('php://stderr', 'w');
```

Comment:
a better solution would be to get the error output from the ConsoleOutputInterface instance